NetCoreRestApi/DataLayer/UnitOfWorks/CategoryUnitOfWork.cs

```diff
@@ -0,0 +1,48 @@
+﻿using DataLayer.Entities;
+using DataLayer.Repositories;
+using DataLayer.UnitOfWorks.Interfaces;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DataLayer.UnitOfWorks
+{
+    public class CategoryUnitOfWork<TId> : ICategoryUnitOfWork<TId>
```

Comment:
fix to class CategoryUnitOfWork : BaseUnitOfWork<CategoryModel, int>, ICategoryUnitOfWork

Reply:
done